data/strings/strings.txt

```diff
@@ -14848,44 +14848,44 @@
   [core_placepage_unknown_place]
     comment = Place Page title for long tap
     tags = android,ios
-    en = Unknown Place
-    af = Onbekende plek
-    ar = مكان غير معروف
-    be = Невядомае месца
-    bg = Неизвестно място
-    ca = Lloc desconegut
-    cs = Neznámé místo
-    da = Ukendt sted
-    de = Unbekannter Ort
-    el = Άγνωστο μέρος
-    es = Lugar desconocido
-    et = Tundmatu koht
-    eu = Toki ezezaguna
-    fa = مکان ناشناس
-    fi = Tuntematon paikka
-    fr = Lieu inconnu
-    hu = Ismeretlen hely
-    id = Tempat Tidak Dikenal
-    it = Luogo sconosciuto
-    ja = 不明な場所
-    ko = 장소 알 수 없음
-    lt = Nežinoma vieta
-    mr = अज्ञात ठिकाण
-    nb = Ukjent sted
-    nl = Onbekende locatie
-    pl = Nieznane miejsce
-    pt = Local desconhecido
-    pt-BR = Local desconhecido
-    ro = Loc necunoscut
-    ru = Неизвестное место
-    sk = Neznáme miesto
-    sv = Okänd plats
-    th = ไม่ทราบชื่อสถานที่
-    tr = Bilinmeyen Yer
-    uk = Невідоме місце
-    vi = Địa Điểm Chưa Biết
-    zh-Hans = 未知位置
-    zh-Hant = 未知的位置
+    en = Map Point
+    af = Kaartpunt
+    ar = نقطة الخريطة
+    be = Пункт карты
+    bg = Точка на картата
+    ca = Punt del mapa
+    cs = Bod na mapě
+    da = Kortpunkt
+    de = Kartenpunkt
+    el = Σημείο χάρτη
+    es = Punto del mapa
+    et = Kaardipunkt
+    eu = Maparen Puntua
```

Comment:
Title Case is unneeded in Basque; this can be "Maparen puntua".

Reply:
Certainly "Maparen puntua" is the correct way.